### ISSUE TITLE
[ACCESSIBILITY] Enhance keyboard accessibility for multiple answer component

### ISSUE THREAD

**Issue by mateuszkulpa** [infermedica/component-library#84]
The multiple answer component with enabled info buttons is not fully accessible for keyboard users. The info buttons cannot be accessed or clicked using keyboard navigation. 

To fix this issue and improve accessibility, the following changes should be implemented:
- Remove the tabindex=-1 attribute from the info button
- Move the info button outside of the radio label element
